app/src/main/java/ua/dancecamp/vmedvediev/ua21summerdancecamp/MyApplication.kt

```diff
@@ -15,16 +17,28 @@ class MyApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
+
         if (BuildConfig.DEBUG) {
```

Comment:
Extract libraries initializations to separate methods like
timber(); 
realm();
stetho();